From ec43dabccae4fc3d5dc482fd2b95c15eff102c0e Mon Sep 17 00:00:00 2001
From: Konsta <konsta09@gmail.com>
Date: Sat, 28 May 2016 19:40:06 +0300
Subject: [PATCH] HACK: intel: ignore media.settings.xml property

* Intel camera HAL passes media.settings.xml property with separate
  media_profiles_*.xml file for both rear and front facing camera (e.g.
  'media.settings.xml=/etc/media_profiles_ov2680b.xml|/etc/media_profiles_ov2680.xml').
* There's no support for parsing several media_profiles_*.xml files
  in libmedia (missing Intel patch) so ignore the property and move
  these configs to unified media_profiles.xml on device side instead.

Change-Id: I1bf7e86b8f287ca9450b0fda72a48867c99e98b8
---
 media/libmedia/MediaProfiles.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/libmedia/MediaProfiles.cpp b/media/libmedia/MediaProfiles.cpp
index 5648a7de753..5cc585987f5 100755
--- a/media/libmedia/MediaProfiles.cpp
+++ b/media/libmedia/MediaProfiles.cpp
@@ -675,7 +675,7 @@ MediaProfiles::getInstance()
     Mutex::Autolock lock(sLock);
     if (!sIsInitialized) {
         char value[PROPERTY_VALUE_MAX];
-        if (property_get("media.settings.xml", value, NULL) <= 0) {
+        if (property_get("media.settings.foo", value, NULL) <= 0) {
             const char *defaultXmlFile = "/etc/media_profiles.xml";
             FILE *fp = fopen(defaultXmlFile, "r");
             if (fp == NULL) {
